app/javascript/src/components/Dashboard/Notes/Pane/Form/index.jsx

```diff
@@ -8,51 +8,87 @@ import { TAGS } from "./constants";
 
 import { NOTES_FORM_VALIDATION_SCHEMA } from "../../constants";
 
-const Form = ({ onClose, refetch, note, isEdit }) => {
-  const handleSubmit = values => {
-    // eslint-disable-next-line no-console
-    console.log(values);
-    refetch;
+const Form = ({ onClose, note, isEdit, setShowPane, notes, setNotes }) => {
+  const handleSubmit = ({ title, description, assignedContact, tag }) => {
+    const updatedNotes = [
+      ...notes,
+      {
+        title,
+        description,
+        assignedContactId: assignedContact.value,
+        tag: tag.value,
+      },
+    ];
+    setNotes(updatedNotes);
+    setShowPane(false);
   };
 
+  const contactList = [
+    {
+      id: 1,
+      name: "Elma Herring",
+      email: "elmaherring@unq.com",
+      phone: "+1 (913) 497-2020",
+    },
+    {
+      id: 2,
+      name: "Knapp Berry",
+      email: "knappberry@unq.com",
+      phone: "+1 (951) 472-2967",
+    },
+  ];
```

Comment:
Why are these constants kept here in the component? If you haven't understood the relevance of a constants file and when and where to use it, then by all means please do clarify your queries. Moving forward I don't expect you to make these kinda mistakes. We document standards in RBP for a reason - to avoid unnecessary reviews like the one I am giving right now.

Reply:
I did move the contacts to constant file after this commit.